internal/authn/oidc/oidc_test.go

```diff
@@ -37,6 +37,7 @@ func TestRemoteOidcAuthenticator_Authenticate(t *testing.T) {
 					"right_issuer",
 					"right_audience",
 					nil,
+					[]string{"openfga client"},
```

Comment:
These tests that you updated can have an empty list of subs and they will still pass